udata_front/theme/gouvfr/datagouv-components/.storybook/preview.tsx

```diff
@@ -4,6 +4,12 @@ import * as React from 'react';
 import { Title, Subtitle, Description, Primary, Controls, Stories } from '@storybook/blocks';
 import { setup, type Preview } from '@storybook/vue3';
 import { setupI18n } from '../src/helpers/i18n';
+import { setupComponents } from "../src/config";
+
+setupComponents({
+  default_lang: "en",
+  only_locales: "en,fr",
```

Comment:
I'm not sure I understand what `only_locales` implies and why not keep default values?

Reply:
only_locales is used for 2 things :
- load files for our date library (dayjs)
- merge locale data for our i18n library

The default value from the components is `en,es,fr`.
We change it in udata-front to only include the current_locale from the metadata.
It could be updated to only `en` because there is no way to change the locale in storybook (a custom development could be added later)